LightFramework/src/rendering/scenerenderer.cpp

```diff
@@ -22,52 +21,97 @@ namespace Light {
 		m_skybox_mesh.reset(VertexArray::create());
 
 		float vertices[] = {
-				//Front
-				-1.0, -1.0, 1.0,
-				1.0, -1.0, 1.0,
-				1.0, 1.0, 1.0,
-				-1.0, 1.0, 1.0,
-				//Left
-				-1.0, -1.0, 1.0,
-				-1.0, 1.0, 1.0,
-				-1.0, 1.0, -1.0,
-				-1.0, -1.0, -1.0,
-				//Right
-				1.0, -1.0, 1.0,
-				1.0, -1.0, -1.0,
-				1.0, 1.0, -1.0,
-				1.0, 1.0, 1.0,
-				//Top
-				-1.0, 1.0, 1.0,
-				1.0, 1.0, 1.0,
-				1.0, 1.0, -1.0,
-				-1.0, 1.0, -1.0,
-				//Bottom
-				-1.0, -1.0, 1.0,
-				-1.0, -1.0, -1.0,
-				1.0, -1.0, -1.0,
-				1.0, -1.0, 1.0,
-				//Back
-				-1.0, -1.0, -1.0,
-				-1.0, 1.0, -1.0,
-				1.0, 1.0, -1.0,
-				1.0, -1.0, -1.0,
+			// Front
+			-1.0,
+			-1.0,
+			1.0,
+			1.0,
+			-1.0,
+			1.0,
+			1.0,
+			1.0,
+			1.0,
+			-1.0,
+			1.0,
+			1.0,
+			// Left
+			-1.0,
+			-1.0,
+			1.0,
+			-1.0,
+			1.0,
+			1.0,
+			-1.0,
+			1.0,
+			-1.0,
+			-1.0,
+			-1.0,
+			-1.0,
+			// Right
+			1.0,
+			-1.0,
+			1.0,
+			1.0,
+			-1.0,
+			-1.0,
+			1.0,
+			1.0,
+			-1.0,
+			1.0,
+			1.0,
+			1.0,
+			// Top
+			-1.0,
+			1.0,
+			1.0,
+			1.0,
+			1.0,
+			1.0,
+			1.0,
+			1.0,
+			-1.0,
+			-1.0,
+			1.0,
+			-1.0,
+			// Bottom
+			-1.0,
+			-1.0,
+			1.0,
+			-1.0,
+			-1.0,
+			-1.0,
+			1.0,
+			-1.0,
+			-1.0,
+			1.0,
+			-1.0,
+			1.0,
+			// Back
+			-1.0,
+			-1.0,
+			-1.0,
+			-1.0,
+			1.0,
+			-1.0,
+			1.0,
+			1.0,
+			-1.0,
+			1.0,
+			-1.0,
+			-1.0,
 		};
 
 		std::shared_ptr<VertexBuffer> vbo(VertexBuffer::create(vertices, sizeof(vertices)));
 
-		vbo->setLayout(BufferLayout({
-			{ ShaderDataType::Float3, "a_Position" }
-		}));
+		vbo->setLayout(BufferLayout({{ShaderDataType::Float3, "a_Position"}}));
```

Comment:
@dropTableUsers42 What do you think of this line? I think it was better the way it was before, minus the spaces after and before the opening and closing curly braces, respectively.

Same for line 137.